content/en/docs/guides/retry.md

```diff
@@ -0,0 +1,129 @@
+  "maxTokens": 10,
+  "tokenRatio": 0.1
+}
+```
+
+For each server, the gRPC client tracks a `token_count` (initially set to `maxTokens`). Failed RPCs decrement the count by 1, successful RPCs increment it by  `tokenRatio`.  If the `token_count` falls below half of `maxTokens`, retries are paused until the count recovers.
+
+Further, hedging is a complementary feature to retries and can be configured similarly. For more details, see the [hedging guide].
+
+### Retry Observability
+
+gRPC supports exposing OpenCensus and OpenTelemetry metrics when retry functionality is enabled. Here's an example of the OpenTelemetry retry attempt statistics available:
+* `grpc.client.attempt.started`
+* `grpc.client.attempt.duration`
+* `grpc.client.attempt.sent_total_compressed_message_size`
+* `grpc.client.attempt.rcvd_total_compressed_message_size`
+
+Metrics at per cal level:
+* `grpc.client.call.duration`
+
+And server side metrics:
+* `grpc.server.call.started`
+* `grpc.server.call.sent_total_compressed_message_size`
+* `grpc.server.call.rcvd_total_compressed_message_size`
+* `grpc.server.call.duration`
+
+Find in-depth metrics and tracing information, along with configuration instructions, in the [gRFC for Otel metrics], [gRFC for retry status].
+
+
+{{% alert title="Transparent Retry" color="info" %}}
+Failure can occur in different stages. Even without an explicit retry policy, gRPC may perform transparent retries. The extent of these retries depends on when the failure happens:
+* gRPC may do unlimited transparent retry when RPC never leaves the client.
+* gRPC perform a single transparent retry when RPC reaches the gRPC server library, but has never been seen by the server application logic. Use caution with this type of retry, as it adds load to the network.
+{{% /alert %}}
+
+### Language guides and examples
+
+| Language | Example          | Documentation        |
+|----------|------------------|----------------------|
+| C++      |                  |                      |
+| Go       | [Go Example]     | 		                   |
+| Java     | [Java Example]   | [Java Documentation] |
+| Python   | [Python Example] |                      | 
+
+
+### Additional Resources
+
+* [gRFC for client side retry]
+* [gRFC for retry status]
+* [Hedging Guide]
+* [gRPC Service Config]
+* [gRFC for Otel metrics]
+
+[gRFC for client side retry]: https://github.com/grpc/proposal/blob/master/A6-client-retries.md  
+[gRFC for retry status]:https://github.com/grpc/proposal/blob/master/A45-retry-stats.md
+[Go Example]: https://github.com/grpc/grpc-go/tree/master/examples/features/retry
+[Java Example]: https://github.com/grpc/grpc-java/tree/master/examples/src/main/java/io/grpc/examples/retrying
+[Python Example]: https://github.com/grpc/grpc/tree/master/examples/python/retry
+[Java Documentation]: https://grpc.github.io/grpc-java/javadoc/io/grpc/ManagedChannelBuilder.html#enableRetry()
+[Hedging Guide]: https://grpc.io/docs/guides/request-hedging
```

Comment:
For links within the site, just use an absolute path so it is considered a link within the site. (e.g., `/docs/guides/request-hedging/`). See https://github.com/grpc/grpc.io/pull/1268